src/main.ts

```diff
@@ -272,7 +273,31 @@ export class MynahUI {
 <li>
 ${item.task ? `<input ${item.checked === true ? 'checked' : ''} disabled type="checkbox">` : ''}
 ${(item.task ? marked.parseInline : marked.parse)(item.text, { breaks: false }) as string}
-</li>`
+</li>`,
+        link: (item) => {
+          // As an additional check, not linkify the pure text links, but keep the []() formatted links
+          let prot;
+          try {
+            prot = decodeURIComponent(unescapeHTML(item.href))
+              .replace(/[^\w:]/g, '')
+              .toLowerCase();
+          } catch (e) {
+            return '';
+          }
+          if (prot.indexOf('javascript:') === 0 || prot.indexOf('vbscript:') === 0 || prot.indexOf('data:') === 0) {
+            return '';
+          }
+
+          if (item.href === item.text && item.title == null) {
```

Comment:
if `item.title` is empty string or undefined, that probably isn't useful to set in `title="..."`, right?
```suggestion
          if (item.href === item.text && !!item.title) {
```